django/db/models/fields/utils.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+
+
+class FieldCacheMixin(object):
+    """
+    Provides a common API for working with the fields
+    value cache on an instance.
+    """
+
+    def get_cache_name(self):
```

Comment:
Do we make use of this indirection layer anywhere to set the cache name to something other than `self.name`? It doesn't look like it. If not, maybe we don't actually need the indirection layer at all?